tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml

```diff
@@ -145,6 +146,21 @@
                 <param name="input_toc_urls" label="Input Total Organic Carbon (TOC)" optional="false" help="URL to in situ Total Organic Carbon (TOC) data (date, value). If the data is divided into parts, please provide several URLs separated by a comma. All URLs need to start with http(s). (URL must be stored in a .txt file)" type="data" format="txt"/>
                 <param name="input_discharge_urls" label="Input discharge" optional="false" help="URL to the daily discharge data (date, value). If the data is divided into parts, please provide several URLs separated by a comma. All URLs need to start with http(s). (URL must be stored in a .txt file)" type="data" format="txt"/>
             </when>
+            <when value="mitgcm-resultplots">
+                <param name="netcdf_input_file" label="NetCDF input file" optional="false" help="The NetCDF input file that should be plotted. It must comply to some specific properties with respect to variables. (URL must be stored in a .txt file)" type="data" format="txt"/>
+                <param name="variable" label="Variable to be plotted" optional="false" help="The variable that should be plotted. You can choose between 'T', 'S', 'U', 'V', 'Eta' and 'W'." type="select">
+                    <option value="T">T</option>
+                    <option value="S">S</option>
+                    <option value="U">U</option>
+                    <option value="V">V</option>
+                    <option value="Eta">Eta</option>
+                    <option value="W">W</option>
+                </param>
+                <param name="plot_time" label="Hour to be plotted" optional="false" help="The time of day (hourly resolution) that should be plotted. Please specify it in this specific format: '20220703-12:00:00'. We assume that the hour is in the simulation period." type="text"/>
```

Comment:
@MarkusKonk all non optional values need to have a default value. This one and the next one.